dashboard/app/models/levels/blockly.rb

```diff
@@ -45,6 +45,7 @@ class Blockly < Level
     ideal
     min_workspace_height
     step_speed
+    start_libraries
```

Comment:
I thought blockly was the CSF levels. Should this go in applab.rb?

Reply:
Thank you for catching that.